docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -521,6 +526,10 @@ As per LogPipeline definition, a dedicated [rewrite_tag](https://docs.fluentbit.
 
 Currently there are the following limitations for LogPipelines that are served by Fluent Bit:
 
+### Unsupported Mode
+
+Usage of a `custom` filter or output enables direct access to the API of the underlying Fluent Bit configuration. As not every feature provided by Fluent Bit can be tested fully, a LogPipeline using custom filters or outputs will run in an `unsupported mode`. Also, full compatibility cannot be guaranteed over releases.
```

Comment:
```suggestion
If you use a `custom` filter or output, you directly access the API of the underlying Fluent Bit configuration. Because Kyma cannot test every feature provided by Fluent Bit, a LogPipeline using custom filters or outputs runs in `unsupported mode`. Also, Kyma cannot guarantee full compatibility over subsequent releases.
```